addons/website/static/src/js/website.contentMenu.js

```diff
@@ -236,6 +238,7 @@ var EditMenuDialog = widget.Dialog.extend({
     add_menu: function () {
         var self = this;
         var dialog = new MenuEntryDialog(this, {menu_link_options: true}, undefined, {});
+        dialog.$modal.addClass('o_contentMenu_link_modal');
```

Comment:
>  So if you only have to style elements which are in the .modal-body [...]

Nope, I need to style `modal-dialog`
                       
`modal` > **modal-dialog** > `modal-content` > **_modal-body_** (affected by XML and JS stuff)